packages/fxa-settings/src/components/FormPassword/index.tsx

```diff
@@ -96,48 +96,22 @@ export const FormPassword = ({
             isSet={formState.dirtyFields.newPassword}
             hasError={errors.newPassword?.types?.length}
           />
-          <Localized id="pw-change-least-8-chars">
-            Must be at least 8 characters
-          </Localized>
+          <Localized id="pw-8-chars">At least 8 characters</Localized>
         </li>
         <li data-testid="change-password-email">
           <ValidationIcon
             isSet={formState.dirtyFields.newPassword}
             hasError={errors.newPassword?.types?.notEmail}
           />
-          <Localized id="pw-change-not-contain-email">
-            Must not be your email address
-          </Localized>
+          <Localized id="pw-not-email">Not your email address</Localized>
         </li>
         <li data-testid="change-password-common">
           <ValidationIcon
             isSet={formState.dirtyFields.newPassword}
             hasError={errors.newPassword?.types?.uncommon}
           />
-          <Localized
-            id="pw-change-common-passwords"
-            elems={{
-              linkExternal: (
-                <LinkExternal
-                  className="link-blue"
-                  data-testid="nav-link-common-passwords"
-                  href="https://support.mozilla.org/en-US/kb/password-strength"
-                >
-                  {' '}
-                </LinkExternal>
-              ),
-            }}
-          >
-            <span>
-              Must not match this{' '}
-              <LinkExternal
-                className="link-blue"
-                data-testid="nav-link-common-passwords"
-                href="https://support.mozilla.org/en-US/kb/password-strength"
-              >
-                list of common passwords
-              </LinkExternal>
-            </span>
+          <Localized id="pw-commonly-used">
+            Not a commonly used password
           </Localized>
         </li>
         <li data-testid="change-password-match">
```

Comment:
Vesta is filing a follow up issue to remove this to match the sign-up flow PW balloon, since making sure your newly entered password matches the confirmation box is not really a "password requirement".